### PR TITLE
714 success notification target points earned points

### LINKED ISSUE #714: Add success notification when Target Points equals Earned points on the side bar
### Overview

When Target points meets earned points display a message notifying the user 

### Action Items

- [x] Add message - TEXT TO BE ADDED BY UI 
- [x]  Only display message when Target points equals earned points
- [ ]  Leave message up for 5 seconds 

### Resources/Instructions

Figma: TBD

Picture: TBD

### PR DESCRIPTION
**Pull request by jahpe777**
Added success notification when Target Points equals Earned points on the side bar closes #714

Screen Shot 2021-02-28 at 6 49 51 PM

### CODE REVIEW

File: client/src/components/ProjectWizard/WizardSidebar/EarnedPointsMetContainer.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+import { createUseStyles, useTheme } from "react-jss";
+
+const useStyles = createUseStyles({
+  pointsNotification: {
+    border: "2px solid transparent",
+    backgroundColor: "#a7c539",
```
Comment (nclairesays) [inline client/src/components/ProjectWizard/WizardSidebar/EarnedPointsMetContainer.js:8]: Can you try using the theme colors here?

File: client/src/components/ProjectWizard/WizardSidebar/EarnedPointsMetContainer.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    padding: "16px 24px",
+    lineHeight: "1.4",
+    margin: "auto"
+  }
+});
+
+const EarnedPointsMetContainer = () => {
+  const theme = useTheme();
+
+  const classes = useStyles({ theme });
```
Comment (nclairesays) [inline client/src/components/ProjectWizard/WizardSidebar/EarnedPointsMetContainer.js:26]: You passed theme in here but didn't actually use it in the createUseStyles

File: client/src/components/ProjectWizard/WizardSidebar/EarnedPointsMetContainer.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  pointsNotification: {
+    border: "2px solid transparent",
+    backgroundColor: "#a7c539",
+    color: "#0F2940",
+    boxShadow: "0px 0px 5px rgba(0, 0, 0, .2)",
+    marginTop: "16px",
+    maxWidth: "250px",
+    display: "flex",
+    justifyContent: "center",
+    cursor: "pointer",
```
Comment (nclairesays) [inline client/src/components/ProjectWizard/WizardSidebar/EarnedPointsMetContainer.js:15]: not sure what this pointer is supposed to signal to the user. pointers usually mean it's clickable but this container is not.

File: client/src/components/ProjectWizard/WizardSidebar/EarnedPointsMetContainer.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+import { createUseStyles, useTheme } from "react-jss";
+
+const useStyles = createUseStyles({
+  pointsNotification: {
+    border: "2px solid transparent",
+    backgroundColor: "#a7c539",
+    color: "#0F2940",
```
Comment (nclairesays) [inline client/src/components/ProjectWizard/WizardSidebar/EarnedPointsMetContainer.js:9]: and here

**Review by nclairesays — CHANGES_REQUESTED** [review]

File: client/src/components/ProjectWizard/WizardSidebar/EarnedPointsMetContainer.js (hunk 1 above)
Comment (jahpe777) [reply client/src/components/ProjectWizard/WizardSidebar/EarnedPointsMetContainer.js]: Added theme colors

File: client/src/components/ProjectWizard/WizardSidebar/EarnedPointsMetContainer.js (hunk 2 above)
Comment (jahpe777) [reply client/src/components/ProjectWizard/WizardSidebar/EarnedPointsMetContainer.js:23]: Using theme now for theme colors

File: client/src/components/ProjectWizard/WizardSidebar/EarnedPointsMetContainer.js (hunk 3 above)
Comment (jahpe777) [reply client/src/components/ProjectWizard/WizardSidebar/EarnedPointsMetContainer.js]: Got rid of pointer

File: client/src/components/ProjectWizard/WizardSidebar/EarnedPointsMetContainer.js (hunk 4 above)
Comment (jahpe777) [reply client/src/components/ProjectWizard/WizardSidebar/EarnedPointsMetContainer.js]: Added theme colors

**Pull request merged**